EIPS/eip-3664.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 3664
+title: ERC-3664 next generation game NFT
+author: TylerZ(@tylerztl), TY(@nimocat)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Simple Summary
+ERC-3664 is a standard for next generation game NFTs, which extends ERC-1155.
+
+## Abstract
+In this proposal, we created gNFT standard, which makes up for the problem that ERC-1155 can only support packed transfer and unchangeable attributes of NFTs. gNFT can make the attributes in the game props have four characteristics: transferable, upgradeable, changeable and evolvable, by using ERC-3664 standard NFTs in game, players are able to complete almost all operations on props only through smart contracts.
+
+## Motivation
+ERC-3664 stems from a new RPG online game we were developed. We found that ERC1155 could not support all operations for game props, such as upgrading, adding attributes, and so on. It is almost impossible for the player to query all the attributes of the NFT on the smart contract, because ERC1155 cannot support the rich properties and functions of the props. For this reason, we propose a new ERC standard: ERC-3664, which is fully compatible with ERC-1155 and can help developers define massive operations on game NFTs. 
```

Comment:
```suggestion
We found that ERC-1155 had limited support for game assets, such as upgrading, adding attributes, and so on. It is almost impossible for the player to query all the attributes of the NFT on the smart contract, because ERC-1155 does not support the rich properties and functions of the assets. For this reason, we propose a new ERC standard which is fully compatible with ERC-1155 and can help developers define massive operations on game NFTs. 
```